### PR TITLE
feat: About page

### PR DESCRIPTION
**Pull request by IgboPharaoh**
This pr adds the About page :
- Adds design, icons and media responsiveness
- Add setup script to fetch reviewer data from [bitcointranscripts](https://api.github.com/repos/bitcointranscripts/bitcointranscripts) repository

The deployment is failing as a GITHUB_ACCESS_TOKEN will be needed to test this implementation.
A further discussion is needed to finalise on this implementation.

### CODE REVIEW

**Comment by Emmanuel-Develops** [comment]
> The deployment is failing as a GITHUB_ACCESS_TOKEN will be needed to test this implementation. 
Added acces token and deployed

File: scripts/fetchReviewers.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const fs = require("fs");
+const path = require("path");
+const dotenv = require("dotenv");
+
+dotenv.config();
+
+const url = `https://api.github.com/repos/bitcointranscripts/bitcointranscripts`;
+const token = process.env.GITHUB_ACCESS_TOKEN;
+
+if (!process.env.GITHUB_ACCESS_TOKEN) {
```
[the remark this answers is not in the document]
Comment (Emmanuel-Develops) [reply scripts/fetchReviewers.js:10]: also add a check for prod.
i.e (!process.env. GITHUB_ACCESS_TOKEN && process.env.NODE_ENV === "production")

having dev contributors generate gh keys to run adds extra restrictions

**Review by Emmanuel-Develops — CHANGES_REQUESTED** [review]
Looks good!

only a couple of changes

File: scripts/fetchReviewers.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      console.log("COMPLETED: fetching commits");
+
+      break;
+    }
+
+    commits.push(...data);
+    page++;
+  }
+
+  const trimText = (text) => (text.length > 15 ? text.substring(0, 15) + "..." : text);
```
[the remark this answers is not in the document]
Comment (IgboPharaoh) [reply scripts/fetchReviewers.js:38]: There is no loss of information here as we are only trimming the string so reviewer title fits into the grid box

File: scripts/fetchReviewers.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  for (const {
+    author: { avatar_url, login },
+  } of commits) {
+    reviewers[login] = { image: avatar_url, title: trimText(login) };
```
[the remark this answers is not in the document]
Comment (IgboPharaoh) [reply scripts/fetchReviewers.js:43]: Its a matter of choice. I am using the the key to add reviewer data to the object which makes the addition on one pass. Using an array I will have to define a condition before it is been added and but for an object, new items with the same key replace existing items with the same key inside the object ensuring every item is unique without setting an extra condition.

File: src/app/about/page.tsx [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+        <Wrapper className='flex flex-col gap-16 md:gap-[72px] max-sm:px-3 py-10 md:py-[104px]'>
+          {/* <GroupedImageSection
+            title='Editors'
+            subText='Editors evaluate and finalize Reviewers submissions, ensuring they’re consistently high quality.'
+            data={editors}
+          /> */}
+
+          <GroupedImageSection
+            title='Reviewers'
+            subText='Editors evaluate and finalize Reviewers submissions, ensuring they’re consistently high quality.'
```
Comment (kouloumos) [inline src/app/about/page.tsx:104]: wrong copy here

**Comment by Emmanuel-Develops** [comment]
Nice work!
The link is linked to a trimmed version of the user github profile (for long usernames).
e.g `https://github.com/Emmanuel-Develo...`, `https://github.com/ai-transcriber[...`, `https://github.com/alexanderwieder...`

which correlates with [this comment](https://github.com/bitcointranscripts/registry/pull/69#discussion_r1878307775)

it helps to have visual changes in the FE while we keep information intact. We can limit the characters with CSS in the component itself. This will be more dynamic and there won't be information loss
cc: @IgboPharaoh

**Comment by Emmanuel-Develops** [comment]
> Great point which I didn't factor in, I've made necessary changes to fix this.

nice. You can also limit with CSS using
```css
    max-width: 100%;
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
```
on the `p` tag.
from the preview deployment, it looks fine without the trim, solely because we do not have very long usernames.
I'm not peculiar on either but to account for future changes, the CSS trim is a good way to go
cc: @IgboPharaoh

**Review by Emmanuel-Develops — APPROVED** [review]

**Comment by Emmanuel-Develops** [comment]
ACK!

@IgboPharaoh this looks good. Merging!

you can complete the dashed linking in a new PR

**Pull request merged**